### PR TITLE
[Feat]: Zoho OAuth Provider Icon

### PR DESCRIPTION
**Pull request by UtkarshAhuja2003**
## What does this PR do?

Adds Zoho Icon for OAuth

## Related PRs and Issues

* https://github.com/appwrite/appwrite/issues/7364

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: static/icons/dark/color/zoho.svg
Comment (stnguyen90) [inline static/icons/dark/color/zoho.svg:1]: Where'd you find this icon? I couldn't find it on their [branding](https://www.zoho.com/branding/) page. 🧐

Comment (UtkarshAhuja2003) [reply static/icons/dark/color/zoho.svg:1]: Got this from [Zoho One](https://www.zoho.com/en-in/one/). I was not able to find a 100x100 Zoho logo😅

Comment (stnguyen90) [reply static/icons/dark/color/zoho.svg:1]: Hmm I'm worried that's a specific product...How about their favicon instead? https://www.zohowebstatic.com/sites/zweb/images/favicon.ico

Comment (UtkarshAhuja2003) [reply static/icons/dark/color/zoho.svg:1]: Done!

**Comment by stnguyen90** [comment]
Dark:
image

Light:
image

File: static/icons/dark/color/zoho.svg
Comment (stnguyen90) [reply static/icons/dark/color/zoho.svg:1]: Sorry, would you please use these?
[zoho_dark_color.zip](https://github.com/appwrite/console/files/14454114/zoho_dark_color.zip)
[zoho_dark_grayscale.zip](https://github.com/appwrite/console/files/14454115/zoho_dark_grayscale.zip)
[zoho_light_grayscale.zip](https://github.com/appwrite/console/files/14454116/zoho_light_grayscale.zip)
[zoho_light_color.zip](https://github.com/appwrite/console/files/14454117/zoho_light_color.zip)

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: static/icons/dark/color/zoho.svg
Comment (UtkarshAhuja2003) [reply static/icons/dark/color/zoho.svg:1]: Updated!! Thanks for the icons 😅

image


image

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**